src/api/gnosisProtocol/hooks.ts

```diff
@@ -34,8 +30,7 @@ type TwapPartOrdersMap = { [twapOrderHash: string]: TwapPartOrderItem }
  */
 export function useGpOrders(account?: string | null, refreshInterval?: number): OrderWithComposableCowInfo[] {
   const { chainId } = useWalletInfo()
-  const emulatedTwapOrders = useAtomValue(emulatedTwapOrdersAtom)
-  const twapParticleOrders = useAtomValue(twapPartOrdersListAtom)
+  const twapPartOrders = useAtomValue(twapPartOrdersAtom)
```

Comment:
Skip changes in this file, just look into https://github.com/cowprotocol/cowswap/pull/2876